commerce_coordinator/apps/commercetools/views.py

```diff
@@ -113,6 +118,8 @@ def post(self, request):
 # noinspection DuplicatedCode
 class OrderSanctionedView(APIView):
     """View to sanction an order and deactivate the lms user"""
+
+    authentication_classes = [JwtBearerAuthentication, SessionAuthentication]
```

Comment:
nit: Just setting the `JwtBearerAuthentication` class is required here.
```suggestion
    authentication_classes = (JwtBearerAuthentication,)
```

Reply:
I tried using only `JwtBearerAuthentication`, but `SessionAuthentication` is required for tests to pass.